_posts/2016-04-25-episerver-developer-meetup-helsinki.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Episerver developer meetup at Helsinki
+author: Rinorragi
+excerpt: What happened in the first Episerver developer meetup held in Helsinki
+categories: 
+- Episerver
+tags: 
+- Episerver 
+- DOTNET 
+- security
+---
+Last week Episerver organized a meetup for Finnish developers in Helsinki. Meetup consisted of pizza, beer and presentations. I loved the meetup and I think that the format was a perfect match to have a cozy atmosphere around the Episerver topic. Only thing that could be done better (for the next time) would be to have more time at the end for the networking! Which just means that the actual event has been great success! 
```

Comment:
"meetup and the format was". I Think on tavallaan turha, koska tavallaan se että nää on sun omia mielipiteitä, goes without saying

...Only thing that **could've** been better (next time maybe? :)

Which just means that the actual event **was** a great